src/afl-forkserver.c

```diff
@@ -58,6 +58,8 @@ static list_t fsrv_list = {.element_prealloc_count = 0};
 
 static void fsrv_exec_child(afl_forkserver_t *fsrv, char **argv) {
 
+  if (fsrv->qemu_mode) setenv("AFL_DISABLE_LLVM_INSTRUMENTATION", "1", 0);
```

Comment:
Please add new env var. to usage()  of afl-clang-fast and the env var. "spell checker".

Reply:
this is an internal env var, and not for users. so it belongs in neither, same as AFL_EINS_ZWEI_POLIZEI